gem/templates/springster/surveys/molo_survey_page.html

```diff
@@ -2,545 +2,223 @@
 {% load wagtailimages_tags wagtailcore_tags static core_tags gem_tags %}
 
 {% block content %}
-{% if self.your_words_competition %}
-  <div class="gcn-competition surveys{{survey.get_effective_extra_style_hints}} surveys--page">
-{% elif self.display_survey_directly %}
-  <div class="polls polls--call-to-action">
-    <div class="polls__bubble-icon"></div>
-    <div class="polls__container">
-{% else %}
-  <div class="surveys surveys{{survey.get_effective_extra_style_hints}} surveys--call-to-action surveys--page">
-    {% if self.image %}
-      {% image self.image width-165 as icon_small %}
-      <img src="{{ icon_small.url }}" alt="{{ self.title }}" class="identity-image__item-icon" />
-    {% else %}
-      <div class="identity-image identity-image--survey"></div>
-    {% endif %}
-{% endif %}
-
-{% if self.your_words_competition %}
-
-  {# include template "patterns/basics/surveys/surveys-modules/surveys_gcn-heading.html" #}
-    {% with pagetype=self %}
-      {% if pagetype.image %}
-          {% image pagetype.image width-98 as tmp_photo_small %}
-          {% image pagetype.image width-320 as tmp_photo_large %}
-          <div class="gcn-competition__image">
-              <img src="{{ tmp_photo_small.url }}" srcset="{{ tmp_photo_large.url }}" alt="{{ survey.image.title }}" class="gcn-competition__dynamic" />
-          </div>
+  <!--HEADING-->
```

Comment:
We're adding more and more comments in our templates, might be a good idea to use django comments `{# comment won't be rendered #}` instead of HTML comments that are viewable on page source